src/components/FAQ-page/Faq.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import Header from "./Header"
+import Question from "./Question"
+import './faq-style.css';
+
+class Faq extends Component {
+   
+    render() {
+      return (
+        <div >
+             <Header />
+             <h1>Frequently Asked Questions</h1>
+             <br />
+             <Question />
+             <br />
+             <Question />
+             <br />
+             <Question />
+             <br />
+             <Question />
+             <br />
+             <Question />
+        </div>
+      );
+    }
+  }
```

Comment:
```
class FAQ extends Component {
  render() {
    const questionText = "Will a beginner, with absolutely no knowledge of github, gain anything fruitful?";
    const answerText = "Yeah,definitely. The organization is meant to assist the beginners grow in the field of development. We’ll have distinct projects appropriate both for beginners as well as the accolades and thereby we’ll make sure that each and every participant gets to learn something new from the projects he or she is contributing for.";

    return (
      <div style={{ display: "flex", flexDirection: "column" }}>
        <Navbar />
        <div style={{ marginTop: 60 }}>
          <h1>Frequently Asked Questions</h1>
          <Question questionText={questionText} answerText={answerText} />
          <Question questionText={questionText} answerText={answerText} />
          <Question questionText={questionText} answerText={answerText} />
          <Question questionText={questionText} answerText={answerText} />
          <Question questionText={questionText} answerText={answerText} />
        </div>
      </div>
    );
  }
}
```

Can we do something like this? Do you think this is a better way to do it?

Reply:
Also rename your component to "FAQ" instead of "Faq" and folder to "FAQ" instead of "Faq-page"